lib/project.ts

```diff
@@ -79,6 +79,18 @@ export class Project implements Project.IProject {
 		}).future<string[]>()();
 	}
 
+	public getConfigFileContent(template: string): IFuture<any> {
+		return (() => {
+			var configFile = _.find(this.projectConfigFiles, configFile => configFile.template === template);
+			if(configFile && this.$fs.exists(configFile.filepath).wait()) {
```

Comment:
it is a bad code practice to use exists and we should remove it from our code base. It is deprecated in both node 0.12 and io.js. See http://nodejs.org/api/fs.html#fs_fs_exists_path_callback .
Better code would be:

```
try {
  var configFileContent = this.$fs.readText(configFile.filepath).wait();
  return configFileContent;
} catch {
  return null;
}
```